builds/jenkins-build-sls-app/Jenkinsfile

```diff
@@ -137,14 +138,10 @@ node() {
   events.sendStartedEvent('CREATE_DEPLOYMENT', 'Serverless application deployment started', environmentDeploymentMetadata.generateDeploymentMap("started", environment_logical_id, gitCommitHash), environment_logical_id)
 
   dir(repo_name) {
-    try {
-      def dd = prepareServerlessYml(config, environment_logical_id) // Generating the deployment descriptor
-      echo "prepareServerlessYml = ${dd}"
-      writeYaml(file: './serverless.yml', data: dd)
-    } catch(FileAlreadyExistsException faee) {
-      echo 'The serverless.yml exists inside the source code thus disregarding the deployment descriptor content from the service catalog'
-      echoServerlessFile()
-    }
+  /* Writing the file from config */
+    def dd = prepareServerlessYml(config, environment_logical_id) // Generating the deployment descriptor
```

Comment:
give meaningful variable name

Reply:
I was hoping that acronym 'dd' that stands for 'Deployment Descriptor' is meaningful enough, but obviously I was mistaken :-). I'll rename shortly

Reply:
Done deal :-)